shuttle/db/db_functions.py

```diff
@@ -111,4 +113,64 @@ def portal_common_profile(username):
         if conn:
             conn.close()
         # if mybethel can't get the data, then prevent anything from loading
-        return abort(503)
\ No newline at end of file
+        return abort(503)
+
+
+def commit_schedule_to_db(table):
+    sql = "DELETE FROM SHUTTLE_SCHEDULE"
+    query(sql, 'write')
+
+    for i in range(len(table)):
+        location = ""
+        for j in range(len(table[i])):
+            try:
+                arrival_time = 0
+                if table[i][j].lower() == 'clc' or table[i][j].lower() == 'anderson' or table[i][j].lower() == 'pine tree' \
+                        or table[i][j].lower() == 'kresge' or table[i][j].lower() == 'scandia' or table[i][j].lower() == 'north':
+                    location = table[i][j].upper()
+                elif table[i][j] == '-' or table[i][j] == 'DROP':
+                    arrival_time = '01-AUG-00 01.00.00.000000000 AM'
+                elif table[i][j].split(':') is not None:
+                    split_time = table[i][j].split(':')
+                    if int(split_time[0]) == 12 or 1 <= int(split_time[0]) < 6:
+                        joined_time = '.'.join(split_time)
+                        arrival_time = '01-SEP-00 ' + joined_time + '.00.000000000 PM'
+                    else:
+                        joined_time = '.'.join(split_time)
+                        arrival_time = '01-SEP-00 ' + joined_time + '.00.000000000 AM'
+                else:
+                    return "data in calendar does not match specified format"
+                if j is not 0:
+                    sql = "INSERT INTO SHUTTLE_SCHEDULE (LOCATION,ARRIVAL_TIME) VALUES (" + "\'" + location + \
+                          "\',\'" + arrival_time + "\')"
+                    query(sql, 'write')
+            except:
+                return "Something went wrong. Please check that the table formatting is correct or call the " \
+                       "ITS Help Desk at 651-638-6500"
+    return "The Calendar has been submitted"
+
+
+def commit_shuttle_request_to_db(location):
+    if location != "":
+        try:
+            now = datetime.datetime.now()
+            date = now.strftime('%d-%b-%Y %I:%M %p')
+            username = flask_session['USERNAME']
+            single_quote = "\'"
+            sql = "INSERT INTO SHUTTLE_REQUEST_LOGS(LOG_DATE,USERNAME,LOCATION,ACTIVE) VALUES (TO_DATE(" + \
```

Comment:
You can remove `ACTIVE` from this INSERT since I made active default to 'Y' when inserting a new row.